### PR TITLE
bump SDK with jpeg & png

### PR DESCRIPTION
**Pull request by PeterPetrik**
- iOS webp support (android had it already)
- jpg support for iOS & Android 
- png support for iOS & Android  (before was only for mbtiles loaded directly from Qt, not GDAL)
- QGIS 3.22.3 final

### CODE REVIEW

**Comment by PeterPetrik** [comment]
@saberraz can you try ?

**Comment by PeterPetrik** [comment]
tests: 
martin/geotiff-jpg
martin/geotiff-webp
Peter.petirk/test_jpg
saber/test_jpg_png_raster

**Comment by saberraz** [comment]
:apple: :100:  working

**Comment by saberraz** [comment]
Android is working well too. Nice one!

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -137,8 +137,9 @@ jobs:
           echo -e "  QMAKE_MACOSX_DEPLOYMENT_TARGET = ${{ env.QMAKE_MACOSX_DEPLOYMENT_TARGET }}"  >> ./input/app/config.pri
           echo -e "  QGIS_QUICK_DATA_PATH = ${{ github.workspace }}/input/app/android/assets/qgis-data"  >> ./input/app/config.pri
           echo -e "  QMAKE_CXXFLAGS += --coverage"  >> ./input/app/config.pri
-          echo -e "  QMAKE_LFLAGS += --coverage "  >> ./input/app/config.pri
-          echo -e "  QMAKE_CXX = ccache \$\$QMAKE_CXX "  >> ./input/app/config.pri
+          echo -e "  QMAKE_LFLAGS += --coverage"  >> ./input/app/config.pri
+          echo -e "  QMAKE_CXX = ccache \$\$QMAKE_CXX"  >> ./input/app/config.pri
+          echo -e "  CONFIG += sdk_no_version_check"  >> ./input/app/config.pri
```
Comment (tomasMizera) [inline .github/workflows/macos.yml:142]: Do we want to include this one? `sdk_check` only prints a warning, no? I wonder if that would not hide some errors in the future

**Review by tomasMizera — COMMENTED** [review]
Good job!

File: .github/workflows/macos.yml (hunk 1 above)
Comment (PeterPetrik) [reply .github/workflows/macos.yml:142]: it is Mac and we do not release it .... and this is only CI, you still have warning on your local build...

Comment (PeterPetrik) [reply .github/workflows/macos.yml:142]: feel free to remove on master if you want, I am easy either way

**Pull request merged**